calour/export_html_template.html

```diff
@@ -2,445 +2,438 @@
 <meta charset="utf-8">
 <html>
 <head>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
-    <script src="http://d3js.org/d3.v4.min.js"></script>
-<style>
-
-.axis path {
-  display: none;
-}
-.pixel {
-  /*image-rendering: pixelated;*/
-  padding: 0px;
-  margin: 0px;
-  border: 0px;
-    image-rendering: optimizeSpeed;             /* STOP SMOOTHING, GIVE ME SPEED  */
-    image-rendering: -moz-crisp-edges;          /* Firefox                        */
-    image-rendering: -o-crisp-edges;            /* Opera                          */
-    image-rendering: -webkit-optimize-contrast; /* Chrome (and eventually Safari) */
-    image-rendering: pixelated; /* Chrome */
-    image-rendering: optimize-contrast;         /* CSS3 Proposed                  */
-    -ms-interpolation-mode: nearest-neighbor;   /* IE8+                           */
-}
-
-.axis line {
-  stroke-opacity: 0.3;
-  shape-rendering: crispEdges;
-}
-
-#reset {
-  position: absolute;
-  top: 20px;
-  left: 20px;
-}
-
-#zoomplus {
-  position: absolute;
-  top: 40px;
-  left: 20px;
-}
-
-#zoomminus {
-  position: absolute;
-  top: 40px;
-  left: 50px;
-}
-
-#moveup {
-  position: absolute;
-  top: 40px;
-  left: 90px;
-}
-
-#movedown {
-  position: absolute;
-  top: 40px;
-  left: 120px;
-}
-
-.line {
-  fill: none;
-  stroke: steelblue;
-  stroke-width: 2px;
-}
-
-</style>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
This is Bootstrap, a very popular web style library. Qiita also uses it as the default. In the formal Qiita plugin I think we don't need this line and the jQuery line, because Qiita already loaded them.